ci-operator/step-registry/baremetal/lab/ipi/conf/network/vips/baremetal-lab-ipi-conf-network-vips-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+echo "Creating patch file to configure ipv4 networking: ${SHARED_DIR}/vips_patch_install_config.yaml"
+
+if [[ "${AGENT_PLATFORM_TYPE}" = "none" ]] || [[ "${masters:-3}" -eq 1 && "${workers:-2}" -eq 0 ]]; then
```

Comment:
Can we update the jobs having master=1 to explicitly use AGENT_PLATFORM_TYPE=none and remove this automated coupling between the number of machines and the platform to use?